microshift_updating/microshift-updating-clusters-overview.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="microshift-updating-clusters-overview"]
+=  Updating {product-title} clusters overview
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-updating-clusters-overview
+
+toc::[]
+
+You can update a {product-title} 4.13 cluster by using the OpenShift CLI (`oc`).
```

Comment:
If you explicitly say "4.13" here, you'll need to update it in subsequent versions (also, isn't there Microshift 4.12 as well?). Would it be better to maybe say something a little more generic, like "You can update a {product-title} cluster by using. . ." and leaving out the current version number? 

This question might be answered below, wherein you say the only upgrade path available is 4.13 to 4.14.

Reply:
Yes, this is an open question. Good idea to remove from here.